src/EditorFeatures/Core/ExternalAccess/VSTypeScript/Api/VSTypeScriptDebugDataTipInfo.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Microsoft.CodeAnalysis.Text;
+
+namespace Microsoft.CodeAnalysis.ExternalAccess.VSTypeScript.Api
+{
+    internal struct VSTypeScriptDebugDataTipInfo
+    {
+        public readonly TextSpan Span;
+        public readonly string Text;
+
+        public VSTypeScriptDebugDataTipInfo(TextSpan span, string text)
+        {
+            Span = span;
+            Text = text;
+        }
+
+        public bool IsDefault
+        {
+            get { return Span.Length == 0 && Span.Start == 0 && Text == null; }
+        }
```

Comment:
This should be updated to follow the wrapper pattern (see other comment for link to example).